index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>CSS Intro</title>
+   
+    <link id="master" rel="stylesheet" href="./master.css">
+    <link id="reset" rel="stylesheet" href="./reset.css">
+
+  </head>
+  <body>
+​
+    <h1>Your Name Here</h1>
+    <div>
+      <p>This is a bunch of information about myself. I'm from here and there and discovered my
+      love of programming when this happend. When I'm not working I'm busy doing this and that.
+    </p>
+    </div>
+​
+    <img src="https://www.breakthrough-pt.com/wp-content/uploads/2014/11/female-default-profile-photo.png" alt="" >
+​
+    <div >
+      <h3>Here are some of my Skills!</h3>
+      <ul>
+        <h3>Languages</h3>
+        <li>JavaScript</li>
+        <li>SQL</li>
+        <li>HTML5</li>
+        <li>CSS3</li>
+​
+      </ul>
+​
+      <ol>
+        <h3>Librarys</h3>
+        <li>React</li>
+        <li>PostgreSQL</li>
+        <li>Node</li>
+        <li>Bootstrap</li>
+​
+      </ol>
+    </div>
+​
+    <div>
+      <h3>Hardest Bug So Far</h3>
+      <p>My hardest bug I ever came across was this infinite loop I couldn't escape. </p>
+      <p>I came up with a totally sick solution though by doing ... </p>
+    </div>
+​
+    <h2>Contact Me</h2>
+    <div>
+     Email me at: <a href="mailto:hello@pursuit.org" target="_top">hello@pursuit.org</a>
+    </div>
+​
+    <ul>
+      <li> <a href="github.com">github link</a> </li>
+      <li> <a href="linkedin.com">LinkedIn link</a> </li>
+      <li> <a href="angellist.com">Angel list link</a> </li>
+    </ul>
+​
+    <form action="index.html" method="post">
+      <input type="text" name="" value="">
+      <input type="submit" name="sumbit" value="submit">
+    </form>
+    <form>    
+      <button>Delete</button>
+      <script>
+      let button = document.querySelector("button");
+          button.addEventListener("click", (reset) => {
```

Comment:
although it is just a variable, it is the convention to name the argument as either `e` or `event`.